### ISSUE TITLE
`schedule:list` shows no configured schedules

### ISSUE THREAD

**Issue by thyseus** [robersonfaria/laravel-database-schedule#14]
Hi, i am using the very most recent laravel version v8.41.0 and unfortunately after installing robersonfaria/laravel-database-schedule the `artisan schedule:list` command is _always_ returning nothing, regardless of what i configure in the backend:

![grafik](https://user-images.githubusercontent.com/654271/117985085-b1f38880-b338-11eb-94d2-e688615b8396.png)

```
# php artisan schedule:list
+---------+----------+-------------+----------+
| Command | Interval | Description | Next Due |
+---------+----------+-------------+----------+
#
```
When removing robersonfaria/laravel-database-schedule again, the command displays the tasks defined in `App\Console\Kernel.php` again as it should. I know this once worked ! Possibly laravel/framework has changed something in recent versions? Can i provide any more debug information ? Or did i forget to register the module somewhere? I am out of clues. Thanks a lot !

**Comment by robersonfaria** [comment]
Sorry for the delay in getting an answer, I went through some troubled days, yesterday I took a calm look at this case and I didn't find an easy way to fix it, anyway I'll keep the issue open and I'll keep looking for a solution.

**Comment by thyseus** [comment]
Hey! Thanks for the reply. Good to know that this issue is real and not some misconfiguration from my side. Perhaps laravel changed something in their way to read schedules? I am quite sure that this extension worked with an ~1 month older laravel version very fine !

**Comment by robersonfaria** [comment]
Hello @thyseus and @andreypopov , I still haven't been able to pay attention to this issue, but I tried to override this command to try to modify its behavior, still without success, I'll try more and if I get something public here.

But that question remains open for us to resolve.

**Comment by robersonfaria** [comment]
Hello @thyseus and @andreypopov , this has been fixed in version 1.1.0
